src/Microsoft.ML.StandardLearners/Standard/MultiClass/Pkpd.cs

```diff
@@ -31,6 +31,29 @@ namespace Microsoft.ML.Runtime.Learners
     using TDistPredictor = IDistPredictorProducing<Float, Float>;
     using CR = RoleMappedSchema.ColumnRole;
 
+    /// <summary>
```

Comment:
>summary [](start = 9, length = 7)

Why here?

Reply:
there's no entry point for it. it is only needed here. 

---
In reply to: [203200111](https://github.com/dotnet/machinelearning/pull/529#discussion_r203200111) [](ancestors = 203200111)